frontend/src/screens/researcher/ScrollToTopButton.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
@divitcr7 take a look at the [mantine affix component](https://mantine.dev/core/affix/)

they have an example there using it for a scroll to top button - we don't need to re-invent the wheel 👍